fedbiomed/researcher/federated_workflows/_federated_workflow.py

```diff
@@ -365,8 +387,42 @@ def info(self, info=None) -> Dict[str, Any]:
             self.experimentation_path(),
             f'- Using: {self._secagg}\n- Active: {self._secagg.active}'
         ]])
+
+        # printing list of items set / not set yet
         print(tabulate.tabulate(info, headers='keys'))
-        return info
+
+        if missing:
+            print("\nWarning: Object not fully defined, missing"
+                  f": \n{missing}")
+        else:
+            print(f"{self.__class__.__name__} can be run now (fully defined)")
+        return info, missing
+
+    def _check_missing_objects(self, missing_objects: Optional[Dict[Any, str]] = None) -> str:
+        """Checks if some objects required for running the `run` method are not set.
+
+        Args:
+            missing_objects: dictionary mapping a string of character
+                naming the required object with the value of the corresponding object
+        """
+        # definitions found missing
+
+        # definitions that may be missing for running the fedreated workflow
+        # (value None == not defined yet for _fds,)
+
+        _not_runable_if_missing = {
+            'Training Data': self._fds,
+            'Tags': self._tags
```

Comment:
Hi @ybouilla, 

I think you can run experiment without `tags`. If you have your training data.

Reply:
I fixed it.